### PR TITLE
Fix forgotten context binding for resetPreviewUrl method

### PR DESCRIPTION
**Pull request by dmitry-pro**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #issuenum
| Related issues/PRs | #issuenum
| License | MIT
| Documentation PR | sulu/sulu-docs#prnum

#### What's in this PR?

There is an error when I try to remove the preview image from a video by clicking `Reset Preview Image` button.
I have made a hotfix for it. Just fixed a method call context and tested it in dev mode.

I have already build it in MediaBundle via `grunt build:js` command, tell me if I forgot something if I did.

image
screenshot 2018-03-03 17 15 39
screenshot 2018-03-03 17 15 51

### CODE REVIEW

**Comment by alexander-schranz** [comment]
Hello @dmitry-pro,

Thank you for your fix can you rebase this fix to the `release/1.5` branch and set the `release/1.5` as target branch.

**Comment by dmitry-pro** [comment]
I cannot figure it out, how can I move my commit into another branch without non-related changes and without re-creating a pull-request for it? I am not too familiar with pull-requests to know how to do it right

**Comment by dmitry-pro** [comment]
Seems, got it :)

**Comment by alexander-schranz** [comment]
@dmitry-pro you need to rebase interactively with `git rebase -i origin/release/1.5` and remove all commits which are not yours and then force push to your branch.

**Comment by dmitry-pro** [comment]
With some messy tricks, got it working

**Comment by alexander-schranz** [comment]
@dmitry-pro thank you! Can you also add a line to the [CHANGELOG.md](https://github.com/sulu/sulu/blob/release/1.5/CHANGELOG.md)

**Comment by dmitry-pro** [comment]
Sure! Just made it

File: src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-edit-overlay/main.js [hunk 1]
```diff
@@ -974,7 +974,7 @@ define([
          */
         toggleDescriptions: function() {
             var checked = this.sandbox.dom.is(
-                    this.sandbox.dom.find(constants.descriptionCheckboxSelector, this.$multiple), ':checked'
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-edit-overlay/main.js]: i think this indentation is correct. could you please reset this change

File: src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-edit-overlay/main.js [hunk 2]
```diff
@@ -989,7 +989,7 @@ define([
          */
         toggleTags: function() {
             var checked = this.sandbox.dom.is(
-                    this.sandbox.dom.find(constants.tagsCheckboxSelector, this.$multiple), ':checked'
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-edit-overlay/main.js]: including this

**Comment by alexander-schranz** [comment]
@wachterjohannes @dmitry-pro did fix that code style.

File: src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-edit-overlay/main.js (hunk 1 above)
Comment (dmitry-pro) [reply src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-edit-overlay/main.js]: Done

**Comment by dmitry-pro** [comment]
Operatively, just done it by myself moment ago :)

**Pull request merged**